reddit_liveupdate/templates/liveupdate.xml

```diff
@@ -1,9 +1,10 @@
 <%!
     from r2.lib.filters import spaceCompress, safemarkdown
```

Comment:
hexdump:

```
00000000  3c 25 21 0a 20 20 20 20  66 72 6f 6d 20 72 32 2e  |<%!.    from r2.|
00000010  6c 69 62 2e 66 69 6c 74  65 72 73 20 69 6d 70 6f  |lib.filters impo|
00000020  72 74 20 73 70 61 63 65  43 6f 6d 70 72 65 73 73  |rt spaceCompress|
00000030  2c 20 73 61 66 65 6d 61  72 6b 64 6f 77 6e 0a 25  |, safemarkdown.%|
00000040  3e 0a 0a 3c 69 74 65 6d  3e 0a 20 20 20 20 3c 67  |>..<item>.    <g|
00000050  75 69 64 20 69 73 50 65  72 6d 61 4c 69 6e 6b 3d  |uid isPermaLink=|
00000060  22 66 61 6c 73 65 22 3e  24 7b 74 68 69 6e 67 2e  |"false">${thing.|
00000070  5f 69 64 7d 3c 2f 67 75  69 64 3e 0a 20 20 20 20  |_id}</guid>.    |
00000080  3c 70 75 62 44 61 74 65  3e 24 7b 74 68 69 6e 67  |<pubDate>${thing|
00000090  2e 5f 64 61 74 65 2e 73  74 72 66 74 69 6d 65 28  |._date.strftime(|
000000a0  27 25 61 2c 20 25 64 20  25 62 20 25 59 20 25 48  |'%a, %d %b %Y %H|
000000b0  3a 25 4d 3a 25 53 20 25  7a 27 29 7d 3c 2f 70 75  |:%M:%S %z')}</pu|
000000c0  62 44 61 74 65 3e 0a 20  20 20 20 3c 64 65 73 63  |bDate>.    <desc|
000000d0  72 69 70 74 69 6f 6e 3e  0a 20 20 20 20 20 20 20  |ription>.       |
000000e0  20 24 7b 73 70 61 63 65  43 6f 6d 70 72 65 73 73  | ${spaceCompress|
000000f0  28 73 61 66 65 6d 61 72  6b 64 6f 77 6e 28 74 68  |(safemarkdown(th|
00000100  69 6e 67 2e 62 6f 64 79  29 29 7d 0a 20 20 20 20  |ing.body))}.    |
00000110  3c 2f 64 65 73 63 72 69  70 74 69 6f 6e 3e 0a 3c  |</description>.<|
00000120  2f 69 74 65 6d 3e 0a                              |/item>.|
00000127

```

looks like a normal space to me.  I think it's the XML syntax highlighting not liking python syntax embedded in XML?